src/sonic-host-services/scripts/procdockerstatsd

```diff
@@ -79,6 +80,8 @@ class ProcDockerStats(daemon_base.DaemonBase):
             value *= 1000
         elif units.lower() == UNITS_MB.lower():
             value *= (1000 * 1000)
+        elif units.lower() == UNITS_GB.lower():
+            value *= (1000 * 1000 * 1000)
```

Comment:
For brevity/readability, I suggest using an exponent here

```python
value *= (1000 ** 3)
```

Would you mind also making similar changes on lines 82, 86 and 88?